rubycritic.gemspec

```diff
@@ -25,6 +25,8 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "flog", "4.3.2"
   spec.add_runtime_dependency "reek", "3.7.1"
   spec.add_runtime_dependency "parser", ">= 2.2.0", "< 3.0"
+  spec.add_runtime_dependency "colorize"
+  spec.add_runtime_dependency "tilt"
```

Comment:
@jbodah can you please address this last comment so we can get this bad boy merged? 
Can't wait to see this live :)